src/extension/layer-filters/src/main/java/au/org/emii/geoserver/extensions/filters/LayerFiltersService.java

```diff
@@ -161,6 +171,19 @@ public void uniqueValues(HttpServletRequest request, HttpServletResponse respons
         }
     }
 
+    private String[] splitLayerName (String layerName) {
+        String[] splitArray = new String[2];
+
+        if(layerName.contains(":") ) {
+            splitArray[0] = layerName.split(":", 2)[0];
```

Comment:
Since split returns an array, it shouldn't be necessary to manually assign elements? Just assign the whole array since it will always have length 2 here?

```java
splitArray = layerName.split(":", 2);
```